server/routes/comments.js

```diff
@@ -5,14 +5,46 @@ import * as dotenv from 'dotenv';
 dotenv.config();
 
 router.get('/', async (req, res) => {
-  const anime_id= req.query.anime_id;
-      try {
+    const anime_id = req.query.anime_id;
+    try {
         const itemList = await db.any(`SELECT comment.*, users.name, users.picture FROM comment JOIN users ON comment.user_id=users.id WHERE anime_id=$1`, [anime_id]);
         res.send(itemList);
-      } catch (e) {
+    } catch (e) {
         console.log(e);
         return res.status(400).json(e);
-      }
-    })
-  
+    }
+})
+
+
+//comment_id , text, user_id , anime_id
+// INSERT INTO comment("Text", "user_id", "anime_id") VALUES('AHHHHH', 1, 322)
+router.post('/', async (req, res) => {
+    const listItem = {
+        //   sub: req.body.user.sub,
+        Text: req.body.Text,
+        sub: req.body.user.sub,
+        anime_id: req.body.anime.anime_id,
+        //   user_id: req.body.user_id
+    }
+    console.log(listItem);
+
+    //select id from users where sub == sub
+    const userSub = `SELECT id FROM users WHERE sub =$1`;
+    const subInfo = await db.query(userSub, [listItem.sub])
+    console.log(subInfo, "Finds respective user id from given sub");
+    const user_id = subInfo[0].id;
+
+    //select anime_id from animelist if it exists
+    const animeId = `SELECT mal_id FROM anime WHERE mal_id=$1 LIMIT 1`;
```

Comment:
Possible Issue: If the user has NOT added this anime to their list there is a chance the anime is not in the database, as a possibility to combat this issye we can nsert to the anime table to grab the anime data to be able to have a valid anime_id

Reply:
Where is the code that inserts an anime into the anime table?

I think I agree that the best thing to do would be to lazy load anime. However, for a first pass, I would probably recommend not doing this and just throwng an error if you can't find the anime in your db. That way you can test the comments feature end-to-end and then go back and fix this edge case.